templates/company/contracts/CompanyTemplate.sol

```diff
@@ -60,6 +60,7 @@ contract CompanyTemplate is BaseTemplate {
         TokenManager tokenManager = _installTokenManagerApp(dao, token, TOKEN_TRANSFERABLE, TOKEN_MAX_PER_ACCOUNT);
         Voting voting = _installVotingApp(dao, token, _voteSettings[1], _voteSettings[2], _voteSettings[0]);
 
+        // Mint tokens
         _mintTokens(acl, tokenManager, _holders, _stakes);
 
         _setupPermissions(dao, acl, agentOrVault, voting, finance, tokenManager, _useAgentAsVault);
```

Comment:
In this template we should remove the overriding `_mintTokens()` definition

Reply:
Yep. I noticed this stray function after we merged the PR and removed it here: https://github.com/aragon/dao-templates/commit/c67e697af8e0b2ea3c9e3654132bf024443bd10b